examples/axisartist/simple_axisline.py

```diff
@@ -25,8 +25,8 @@
 ax.set_xlabel("Label X")
 ax.set_ylabel("Label Y")
 # or
-#ax.axis["bottom"].label.set_text("Label X")
-#ax.axis["left"].label.set_text("Label Y")
+#     ax.axis["bottom"].label.set_text("Label X")
+#     ax.axis["left"].label.set_text("Label Y")
```

Comment:
```suggestion
# ax.axis["bottom"].label.set_text("Label X")
# ax.axis["left"].label.set_text("Label Y")
```
I wouldn't use extra indentation for alternative code. It lives semantically at the level of ` #` and thus should follow with only one space for readability. That's also what some editors do for their builtin comment/uncomment commands. Same holds for some places below.